addOns/pscanrules/src/main/java/org/zaproxy/zap/extension/pscanrules/StrictTransportSecurityScanRule.java

```diff
@@ -263,4 +274,18 @@ private String getMetaHSTSEvidence(Source source) {
         }
         return null;
     }
+
+    @Override
+    public List<Alert> getExampleAlerts() {
+        List<Alert> alerts = new ArrayList<>();
```

Comment:
Could still use List.of, or if the evidence is moved to the enum it could just iterate the values.